README.md

```diff
@@ -1,6 +1,6 @@
 DarkRP Leveling System
 ======================
-For DarkRP 2.5.0 and above. Adds leveling functionality and allows you to limit item purchasing based on a players level.
+For DarkRP 2.7.0 and above. Adds leveling functionality and allows you to limit item purchasing based on a players level.
```

Comment:
I think it should still work on 2.5.0?

Reply:
i don't have previous darkrp version so idk if it works